src/commands/registries/azure/deployImageToAzure.ts

```diff
@@ -62,11 +62,19 @@ export async function deployImageToAzure(context: IActionContext, node?: RemoteT
     await wizard.execute();
 
     const site: Site = nonNullProp(wizardContext, 'site');
-    const createdNewWebApp: string = `Successfully created web app "${site.name}": https://${site.defaultHostName}`;
+    const siteUri: string = `https://${site.defaultHostName}`;
+    const createdNewWebApp: string = `Successfully created web app "${site.name}": ${siteUri}`;
     ext.outputChannel.appendLine(createdNewWebApp);
+
+    const openLink: string = 'Open Link';
```

Comment:
Perhaps `'Open Site'` or something more specific than "link"?

Reply:
Thanks @philliphoff, 'Open Site' is much better.